packages/react-ui-core/src/Gmap/Gmap.js

```diff
@@ -1,67 +1,87 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
-import themed from 'react-themed'
+import autobind from 'autobind-decorator'
 import { parseArgs } from '@rentpath/react-ui-utils'
 import GmapBase from './GmapBase'
 
-const createComponent = (element, className) => {
-  if (React.isValidElement(element)) return element
-
-  const [Container, props] = parseArgs(element, 'div', { className })
-  return <Container {...props} />
-}
-
-@themed(/^Gmap/, { pure: true })
+const API_BASE_URL = 'https://maps.googleapis.com/maps/api/js'
+const LIBRARIES = []
 
 export default class Gmap extends PureComponent {
   static propTypes = {
-    loadingElement: PropTypes.oneOfType([
-      PropTypes.object,
-      PropTypes.element,
+    apiKey: PropTypes.string.isRequired,
+    libraries: PropTypes.arrayOf(
+      PropTypes.string,
+    ),
+    spinner: PropTypes.oneOfType([
+      PropTypes.func,
       PropTypes.node,
-    ]),
-    containerElement: PropTypes.oneOfType([
       PropTypes.object,
-      PropTypes.element,
     ]),
-    mapElement: PropTypes.oneOfType([
-      PropTypes.object,
-      PropTypes.element,
-    ]),
-    theme: PropTypes.object,
   }
 
   static defaultProps = {
-    theme: {},
+    libraries: LIBRARIES,
```

Comment:
Just curious, why didn't you just set the default libraries to `[]`?

Reply:
Good question, I think it was because I actually had something in it originally.  I'll switch it.